src/components/rangeslider/attributes.js

```diff
@@ -0,0 +1,48 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var colorAttributes = require('../color/attributes');
+
+module.exports = {
+    visible: {
+        valType: 'boolean',
+        dflt: false,
+        description: 'Determines whether or not the range slider will be visible.'
+    },
+    bordercolor: {
+        valType: 'color',
+        dflt: colorAttributes.defaultLine,
+        role: 'style',
+        description: 'Sets the border color of the range slider.'
+    },
+    borderwidth: {
+        valType: 'integer',
+        dflt: 0,
+        role: 'style',
+        description: 'Sets the border color of the range slider.'
+    },
+    backgroundcolor: {
```

Comment:
this should be `bgcolor` for consistency with legend, annotations, colorbar, scene attributes.